python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -222,7 +260,15 @@ class Registry(object):
     """
 
     def __init__(self, name, doc, dirty_flush_counter=10, update_index_time=30, dirty_max_timeout=60, dirty_min_timeout=30):
-        """Registry constructor, giving public name and documentation"""
+        """Registry constructor, giving public name and documentation
+        Args:
+            name (str): Name of teh registry e.g. 'jobs', 'box', 'prep'
+            doc (str): This is the doc string of the registry describing it's use and contents
+            dirty_flush_counter (int): This is no longer used
+            update_index_time (int): This is no longer used
+            dirty_max_timeout (int): This is no longer used
+            dirty_min_timeout (int): This is no longer used
```

Comment:
Aside from my comments on #415 OK. Can you poke me when that finishes as we're going to have to wait for the tests to re-run several times before I can get to work on merging the changes into this PR.